monitor/keys.go

```diff
@@ -12,8 +15,10 @@ var (
 )
 
 // get puller key from mainchain txHash
-func GetPullerKey(txHash mainchain.HashType) []byte {
-	return append(PullerKeyPrefix, txHash.Bytes()...)
+func GetPullerKey(eLog ethtypes.Log) []byte {
```

Comment:
puller?

Reply:
This still for the puller functionalities

Reply:
ok